src/interaction/iodevices.jl

```diff
@@ -4,206 +4,205 @@ represent keyboard buttons.
 """
 module Keyboard
```

Comment:
I don't think we should lose the indention for modules with how we use them here.